data_files/ZOS_Software_List.json

```diff
@@ -233,5 +237,85 @@
         "packageName":"zTron",
         "description":"Ambitus hosted lightweight pipeline framework for creating Python workflows as standalone application and/or Jupyter notebooks",
         "version":""
+    },
+    {
+        "packageName":"zOS-Archetypes",
+        "description":"A \"z/OS Archetype\" defines a fundamental role that is commonly associated with the z/OS computing platform. https://github.com/IBM/IBM-Z-zOS/tree/main/zOS-Archetypes",
+        "version":""
+    },
+    {
+        "packageName":"zOS Automation",
+        "description":"This directory is intended for tutorials, samples, and other useful information around IBM System Automation for z/OS and NetView for z/OS. https://github.com/IBM/IBM-Z-zOS/tree/main/zOS-Automation",
+        "version":""
+    },
+    {
+        "packageName":"zOS-CPM",
+        "description":"This repository contains CPM resources that might be useful to the z/OS community. https://github.com/IBM/IBM-Z-zOS/tree/main/zOS-CPM",
+        "version":""
+    },
+    {
+        "packageName":"zOS Data Gatherer",
+        "description":"This repository contains z/OS Data Gatherer resources that might be useful to the z/OS community. https://github.com/IBM/IBM-Z-zOS/tree/main/zOS-DataGatherer",
+        "version":""
+    },
+    {
+        "packageName":"zOS-Education",
+        "description":"This repository contains IBM® Education Assistance modules that you might find to be useful in learning about z/OS®. https://github.com/IBM/IBM-Z-zOS/tree/main/zOS-Education",
```

Comment:
2 ® symbols here.

Reply:
> Turns out, the tool does not like the ® symbol in the JSON, so it fails to load in the tool when that's included. I've noted each instance of it in the file, and you can just go ahead and remove them and it should be safe to merge.
> 
> Tip: If you have a local instance of Software Discovery Tool running, load up the file in it to confirm it's working in your instance to catch these sorts of issues. In my test, it caused the cached_data.json file to not be created because it was failing on the ® symbols (though it did it silently, so it took some figuring out!).

Got it! I will fix this immediately

Reply:
@pleia2 done!